### ISSUE TITLE
Reconstruction loss converges in Autoencoders (embedding  projected into UMAP), but not in Parametric UMAP

### ISSUE THREAD

**Issue by gg4u** [lmcinnes/umap#1037]
Can you please understand :
- range of UMAP loss
- challenges in reconstruction losses between training Autoencoders as components of Parametric UMAP, VS AE models trained indepentently and then projected.
 
Below a description to help clarify the concepts:
---

Please look at the plot of the losses in :
https://github.com/lmcinnes/umap/blob/a7606f2d413c278cc98d932de62f829914209c4f/notebooks/Parametric_UMAP/04.0-parametric-umap-mnist-embedding-convnet-with-autoencoder-loss.ipynb#L211
```
ax.plot(embedder._history['loss']) # UMAP loss 

ax.plot(embedder._history['reconstruction_loss'], label='train')  # Autoencoder loss => the MSE
ax.plot(embedder._history['val_reconstruction_loss'], label='valid')
```

<img width="634" alt="Screenshot 2023-08-01 at 12 29 05 PM" src="https://github.com/lmcinnes/umap/assets/284214/92b488f4-8835-4372-8e39-51abd9fbc5f3">

1.
Is the y-axis of first plot a Cross Entropy ? I thought that would refer to the loss for computing the manifold embedding - UMAP loss. The values are also diffrent betwween the two plots, so I guess two different thigns
2. which is the metric of UMAP Loss ? 
3. Can you please better understand comparison of using Cross Entropy VS MSE in Parametric UMAP, and how affects learning in your experience ?

When training an Autoencoder by itself (MSE loss), it trains very fast. When training as part of the Parametric UMAP, it is very slow (same model). Parametric UMAP seems to use BinaryCross Entropy.
In the original paper there is also a comparison (see below), but the comparison is beween a model trained with MSE loss, and a Parametric UMAP trained with Cross Entropy (using the same model).

Why not comparing Parametric UMAP trained with MSE ? Any reasons for doing that ?
Do you strongly advice to just use default Cross Entropy and scale all inputs [0,1] ?
(I have been using MSE with [-1;1])

Thanks very much for helping in understanding key concepts but also best practices!!!

https://arxiv.org/pdf/2009.12981.pdf
```
<img width="649" alt="Screenshot 2023-08-01 at 12 38 42 PM" src="https://github.com/lmcinnes/umap/assets/284214/20184e54-12b1-42fe-85be-46816f68e751">

```
